mock_bin/echoboth.rs

```diff
@@ -0,0 +1,10 @@
+fn main() {
+    let mut space = "";
+    for arg in std::env::args().skip(1) {
+        eprint!("{}{}", space, arg);
+        print!("{}{}", space, arg);
+        space = " ";
+    }
+    eprintln!();
+    println!();
+}
```

Comment:
There's `mock_bin/echoboth` binary in the repo now. Let's add it to the .gitignore, and rewrite git history to make sure there isn't a commit with large binary file.